rfc/0054-sdk3-management-apis.md

```diff
@@ -3068,7 +3073,49 @@ Nothing
 
 ### URI
 
-    POST http://localhost:8091/pools/default/buckets/<bucket>/collections/<scope_name> -d name=<collection_name> -d maxTTL=<maxExpiry>
+    POST http://localhost:8091/pools/default/buckets/<bucket>/scopes/<scope_name> -d name=<collection_name> -d maxTTL=<maxExpiry> -d history=<history>
```

Comment:
Is this correct?  In JVM world we are using /pools/default/buckets/{bucketName}/scopes/{scopeName}/collections (which also matches the PATCH path below)